src/link-manager.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * @fileoverview Link manager for controlling the link query.
+ */
+
+/** @const */
+genotet.linkManager = {};
+
+/**
+ * Object that stores the link views by their type.
+ */
+/** @type {!Object<!Object<!Array<{
+ *    target: genotet.View,
+ *    response: string
+ *  }>>>}
+ */
+genotet.linkManager.links = {};
+
+/** @const {!Array<string>} */
+genotet.linkManager.EXPRESSION_ACTIONS = ['addProfile'];
+
+/** @const {!Array<string>} */
+genotet.linkManager.NETWORK_ACTIONS = ['nodeClick', 'edgeClick'];
+
+/** @const {!Array<string>} */
+genotet.linkManager.BINDING_ACTIONS = ['updateTrack', 'locus'];
+
+
+/**
+ * Settings for link views.
+ * This part will be extended to user self-defined in next version.
+ * @const {!Array<!Object<{
+ *    source: string,
+ *    action: string,
+ *    target: string,
+ *    response: string
+ *  }>>}
+ * */
+genotet.linkManager.LINK_SETTINGS = [
```

Comment:
I agree with that different presets need to have different links. This would be user-defined in next version, so that I think it's better to keep them in variables than in a dedicated file. Now we should use constants to replace the variables.